x/incentives/types/expected_keepers.go

```diff
@@ -16,19 +16,14 @@ type BankKeeper interface {
 
 	HasSupply(ctx sdk.Context, denom string) bool
 
-	SendCoinsFromModuleToAccount(
-		ctx sdk.Context, senderModule string, recipientAddr sdk.AccAddress, amt sdk.Coins,
-	) error
+	SendCoinsFromModuleToAccount(ctx sdk.Context, senderModule string, recipientAddr sdk.AccAddress, amt sdk.Coins) error
```

Comment:
I would try using `InputOutputCoins` here if possible. 
This way, there can be a single input (one module account balance update equivalent to the sum of all locks rewards) and `n` outputs, instead of `n` inputs and `n` outputs, as long as the `sum(inputs)=sum(outputs)`. 
So,  instead of `n+n`, you would end up with `n+1` account balance updates.

Reply:
I think for this use-case, it would be a good fit, I believe this scenario would be adequate for its intended purpose. Also, gas would be lower probably